imageregistry/v1/types.go

```diff
@@ -10,6 +10,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ConfigList is a slice of Config objects.
+// +openshift:compatibility-gen:level=1
```

Comment:
level 4, exposed=false and make a note to revisit with the image team as a followup to figure  out if users touch this.